swift/swift.py

```diff
@@ -204,50 +227,58 @@ def _swiftcall(self, msg: str):
+        
+        Raises:
+            RuntimeError: When the user rejects the request.
+        
+        Returns:
+            The returned value of the swift-call, if any.
         """
-        try:
-            msg = json.loads(msg)
-        except json.JSONDecodeError as e:
-            print(f"swift.swift._swiftcall(): {e!r}")
-            return
-        if any(key not in msg for key in ("action", "args")):
-            print("The message was ignored because "
-                  "it has no such key; action or args.")
-            return
+        msg = json.loads(msg)
         action, args = msg["action"], msg["args"]
         if action == "create":
-            if any(key not in args for key in ("name", "info")):
-                print("The message was ignored because "
-                      "args of the create action have no such key; name or info.")
-                return
             name, info = args["name"], args["info"]
             reply = QMessageBox.warning(
                 None,
                 "swift-call",
-                f"The app {self.sender().name} requests for creating an app {name}",
+                f"The app {sender} requests for creating an app {name}",
                 QMessageBox.Ok | QMessageBox.Cancel,
                 QMessageBox.Cancel
             )
             if reply == QMessageBox.Ok:
-                self.createApp(name, AppInfo(**info))
-        elif action == "destroy":
-            if any(key not in args for key in ("name",)):
-                print("The message was ignored because "
-                      "args of the destroy action have no such key; name.")
-                return
+                return self.createApp(name, AppInfo(**info))
+            raise RuntimeError("user rejected the request.")
+        if action == "destroy":
             name = args["name"]
             reply = QMessageBox.warning(
                 None,
                 "swift-call",
-                f"The app {self.sender().name} requests for destroying an app {name}",
+                f"The app {sender} requests for destroying an app {name}",
                 QMessageBox.Ok | QMessageBox.Cancel,
                 QMessageBox.Cancel
             )
             if reply == QMessageBox.Ok:
-                self.destroyApp(name)
+                return self.destroyApp(name)
+            raise RuntimeError("user rejected the request.")
+        raise NotImplementedError(action)
+
+    def _swiftcall(self, sender: str, msg: str):
+        """Will be connected to the swiftcallRequested signal.
+
+        Note that swiftcallRequested signal only has one str argument.
+        In fact the partial method will be connected using functools.partial().
+
+        Args:
+            sender: See _handleSwiftcall().
+            msg: See _handleSwiftcall().
+        """
+        try:
+            value = self._handleSwiftcall(sender, msg)
+        except Exception as error: # pylint: disable=broad-exception-caught

```

Comment:
Suppressing the warning looks good!

But, please start at least 2 spaces from the code. It is referred to [Google Python Style Guide](https://google.github.io/styleguide/pyguide.html#385-block-and-inline-comments).